examples/link_baseline.py

```diff
@@ -0,0 +1,108 @@
+import argparse
+import random
+from collections import Counter
+
+import duckdb
+import numpy as np
+import pandas as pd
+
+from relbench.datasets import get_dataset
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--dataset", type=str, default="rel-stackex")
+parser.add_argument("--task", type=str, default="rel-stackex-comment-on-post")
+parser.add_argument("--repeats", type=int, default=1)
+parser.add_argument("--method", type=str, default="most_frequent")
+args = parser.parse_args()
+
+dataset = get_dataset(name=args.dataset, process=True)
+task = dataset.get_task(args.task, process=True)
+
+train = task.train_table.df
+test = task.test_table.df
+PAD = -1
```

Comment:
I think we should use the train+val to make test prediction. That aligns better with our GNN models. Also, can we follow the format [here](https://github.com/snap-stanford/relbench/blob/main/examples/baseline.py)?